assets/js/amp-block-validation.js

```diff
@@ -249,16 +249,20 @@ var ampBlockValidation = ( function() { // eslint-disable-line no-unused-vars
 				noticeMessage += wp.i18n.__( 'Non-accepted validation errors prevent AMP from being served, and the user will be redirected to the non-AMP version.', 'amp' );
 			}
 
-			noticeElement = wp.element.createElement( 'p', {}, [
-				noticeMessage + ' ',
-				ampValidity.review_link && wp.element.createElement(
-					'a',
-					{ key: 'review_link', href: ampValidity.review_link, target: '_blank' },
-					wp.i18n.__( 'Review issues', 'amp' )
-				)
-			] );
+			noticeOptions = {
+				id: 'amp-errors-notice'
+			};
+			if ( ampValidity.review_link ) {
+				noticeOptions.actions = [
+					{
+						label: wp.i18n.__( 'Review issues', 'amp' ),
+						url: ampValidity.review_link
+					}
+				];
+			}
 
-			module.validationWarningNoticeId = wp.data.dispatch( 'core/editor' ).createWarningNotice( noticeElement, { spokenMessage: noticeMessage } ).notice.id;
+			wp.data.dispatch( 'core/notices' ).createNotice( 'warning', noticeMessage, noticeOptions );
```

Comment:
I'm getting an error here now when attempting to edit an existing post:

![image](https://user-images.githubusercontent.com/134745/48669392-95a06900-eab8-11e8-9e7b-a1e6f178b5d1.png)

Reply:
@westonruter Weird, that would indicate as if `core/notices` wouldn't exist. Is this with the latest version of Gutenberg?

Reply:
Humm. I'm not seeing the issue anymore. I opened an incognito window and everything. I'll report if I see it again.